seminar3/3-1.py

```diff
@@ -0,0 +1,18 @@
+#Задача 16: Требуется вычислить, сколько раз встречается некоторое число X в массиве A[1..N]. Пользователь
+#в первой строке вводит натуральное число N – количество элементов в массиве. В последующих  строках записаны N целых чисел Ai. Последняя строка содержит число X
+
+from random import randint
+
+# создание и вывод массива
+
+n = int(input('Введите размер массива '))
+array=[0]*n
+for i in range(n):
+    array[i]=randint(0,20)
```

Comment:
Проще и быстрее использовать метод `append`
```suggestion
array=[]
for i in range(n):
    array.append(randint(0, 20))
```